python36Packages.plyvel: 0.9 -> 1.0.5

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/python3.6-plyvel/versions.



meta.description for python36Packages.plyvel is: '"Fast and feature-rich Python interface to LevelDB"'.


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 1.0.5 with grep in /nix/store/2hzzx25biwfxphbb9hck6lfzhzdnjnbr-python3.6-plyvel-1.0.5
- directory tree listing: https://gist.github.com/c303c1f5ef12d1540f16687c5b6b60a7
- du listing: https://gist.github.com/489658fc846f1188bc554f7e12ac78e8

</details>
<details>
<summary>
Outpath report (click to expand)
</summary>

Outpath difference report
11 total rebuild paths

3 package rebuilds

3 x86_64-linux rebuilds
3 i686-linux rebuilds
2 x86_64-darwin rebuilds
3 aarch64-linux rebuilds


First ten rebuilds by attrpath
python27Packages.plyvel
python36Packages.plyvel
tribler

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/2hzzx25biwfxphbb9hck6lfzhzdnjnbr-python3.6-plyvel-1.0.5 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A python36Packages.plyvel https://github.com/r-ryantm/nixpkgs/archive/a117531c9aa0829b82572aa586d7eb387f1f30ac.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/2hzzx25biwfxphbb9hck6lfzhzdnjnbr-python3.6-plyvel-1.0.5
ls -la /nix/store/2hzzx25biwfxphbb9hck6lfzhzdnjnbr-python3.6-plyvel-1.0.5/bin
```


</details>
<br/>

r-ryantm committed:
python36Packages.plyvel: 0.9 -> 1.0.5

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.6-plyvel/versions

GrahamcOfBorg commented:
<!--REQUEST_ID=dad29967-a427-4aa9-bba2-4acd94c4f624-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49987&attempt_id=c8106761-1c6a-4867-88e0-cdb929b14712)

Attempted: python36Packages.plyvel

<details><summary>Partial log (click to expand)</summary><p>

```


----------------------------------------------------------------------
Ran 3 tests in 0.001s

FAILED (errors=3)
Test failed: <unittest.runner.TextTestResult run=3 errors=3 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=3 errors=3 failures=0>
builder for '/nix/store/s6szf4nbb6p40vbjrn2swqx0gqrm163h-python3.6-plyvel-1.0.5.drv' failed with exit code 1
error: build of '/nix/store/s6szf4nbb6p40vbjrn2swqx0gqrm163h-python3.6-plyvel-1.0.5.drv' failed
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=dad29967-a427-4aa9-bba2-4acd94c4f624-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49987&attempt_id=09e53701-fe8f-4cdc-946b-121fe67e28eb)

Attempted: python36Packages.plyvel

<details><summary>Partial log (click to expand)</summary><p>

```
writing manifest file 'plyvel.egg-info/SOURCES.txt'
running build_ext
copying build/lib.linux-aarch64-3.6/plyvel/_plyvel.cpython-36m-aarch64-linux-gnu.so -> plyvel

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
pytestcachePhase
/nix/store/ppn0wvcf2f9j9y44mx7brjrqbjj41yg9-python3.6-plyvel-1.0.5
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=dad29967-a427-4aa9-bba2-4acd94c4f624-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49987&attempt_id=fd466166-0b5e-489a-b324-1845bfae793c)

Attempted: python36Packages.plyvel

<details><summary>Partial log (click to expand)</summary><p>

```
writing manifest file 'plyvel.egg-info/SOURCES.txt'
running build_ext
copying build/lib.linux-x86_64-3.6/plyvel/_plyvel.cpython-36m-x86_64-linux-gnu.so -> plyvel

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
pytestcachePhase
/nix/store/2hzzx25biwfxphbb9hck6lfzhzdnjnbr-python3.6-plyvel-1.0.5
```
</p></details>

Mic92 committed:
plyvel: restrict to linux

on macOS we see the following error:

ImportError: Failed to import test module: test.test_plyvel
Traceback (most recent call last):
  File "/nix/store/0mha803yql9q7b6h93m8940z3sjbhiy8-python3-3.6.6/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/nix/store/0mha803yql9q7b6h93m8940z3sjbhiy8-python3-3.6.6/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/private/tmp/nix-build-python3.6-plyvel-1.0.5.drv-0/plyvel-1.0.5/test/test_plyvel.py", line 17, in <module>
    import plyvel
  File "/private/tmp/nix-build-python3.6-plyvel-1.0.5.drv-0/plyvel-1.0.5/plyvel/__init__.py", line 6, in <module>
    from ._plyvel import (  # noqa
ImportError: dlopen(/private/tmp/nix-build-python3.6-plyvel-1.0.5.drv-0/plyvel-1.0.5/plyvel/_plyvel.cpython-36m-darwin.so, 2): Library not loaded: /nix/store/rq39v3s1l6vc85wwmrxsmlpi4rnbjnx6-leveldb-1.20/lib/out-shared/libleveldb.dylib.1.20
  Referenced from: /private/tmp/nix-build-python3.6-plyvel-1.0.5.drv-0/plyvel-1.0.5/plyvel/_plyvel.cpython-36m-darwin.so
  Reason: image not found
